package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.2.0",
   "private": false,
   "dependencies": {
+    "@patternfly/patternfly-next": "^1.0.44",
     "@patternfly/react-core": "1.11.1",
     "@red-hat-insights/insights-frontend-components": "^3.14.3",
```

Comment:
Might as well bump this to `^3.14.4`

Reply:
fine. but npm didn't tell me to, but **fine**.